src/contributor-guide/contributing_dod.md

```diff
@@ -14,32 +14,31 @@ The requirements that result from our DoD are applied by default to all Magento-
 Internally, the Magento development teams follow the DoD to ensure that all work meets consistent release criteria.
 We encourage our community contributors to also follow the Magento DoD.
 
-At a high level, the criteria defined in our "definition of done" revolve around the principle of writing clean code that is reviewed and includes unit tests and solid documentation along with the code.
+At a high level, the criteria defined in our "definition of done" revolve around the principle of writing clean code that is reviewed and includes automated tests and solid documentation along with the code.
 
-The following sections provide additional details about each of these criteria:
+## Definition of Done
 
-## Backward Compatibility
+The checklist that can be used to evaluate if the DoD has been met:
 
-### Functional Backward Compatibility
+-  All changes must be approved by reviewer.
+-  All changes must be covered by automated tests if that is possible.
```

Comment:
Manual test cases should be created to cover scenarios where automation is not possible.

Reply:
I did not change the content of DoD apart from TCO analysis and static tests coverage for BIC topics, just decomposition it into several pages for better readability.

Adding manual test cases requirement to DoD should be discussed in a separate PR.